src/components/Collection/SidebarAboutTab.js

```diff
@@ -2,31 +2,95 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { chopString } from '../../services/helpers';
 import { getESDescription } from '../../services/elasticsearch-parser';
+import { Link } from 'react-router-dom';
+import { facetValues } from '../../services/reactive-search';
+
+const AboutMetaValues = props => {
+  const values = [...props.items];
+
+  // Facet value provided, so give a link to the Search Results page to prep the search results
+  if (props.facetValue) {
+    return values.map(value => (
+      <li key={value}>
+        <Link
+          to={{
+            pathname: '/search',
+            state: {
+              facetValue: props.facetValue,
+              searchValue: value
```

Comment:
If it's going to hook into that date range slider we'd need to send the value of the `year` property vs. the `date` property. Or is this just going to link to that value for search text? I can't quite tell what's going on here. 


![screen shot 2018-12-04 at 1 23 31 pm](https://user-images.githubusercontent.com/6372022/49470192-fe812400-f7ce-11e8-966e-502169963c60.png)

![screen shot 2018-12-04 at 1 23 25 pm](https://user-images.githubusercontent.com/6372022/49470224-093bb900-f7cf-11e8-8b7e-0c9c4298a765.png)

Reply:
Yea, I'm not sure Date needs to be a clickable search filter or not.  Thoughts?  For now I could just display the list of user entered dates here, and have it not be clickable?   It's not clickable from the Item Details page metadata, from what I recall.

Reply:
I think not clickable is fine for now. Whatever's easiest